### PR TITLE
Do you believe in magic (methods)?

### PR DESCRIPTION
**Pull request by craigjmidwinter**

### CODE REVIEW

File: example.py [hunk 1]
```diff
@@ -1,5 +1,6 @@
 from pyowlet.PyOwlet import PyOwlet
 import sys
+import time
 from pprint import pprint
 
 if len(sys.argv) != 3:
```
Comment (angel12) [inline example.py:6]: Line 7 needs to be "example.py" not "test.py"

File: example.py [hunk 2]
```diff
@@ -12,6 +13,19 @@
...
+    print('These properties will fetch new information automatically if the information is older than 15 seconds.\n')
+    print('You can also bypass the ratelimiting and refresh the properties manually\n')
+
+    pyowletClient.update_properties()
+
+    print('Or query for raw measurements individually\n')
+
     properties = [
         'OXYGEN_LEVEL',
         'HEART_RATE',
```
Comment (angel12) [inline example.py:31]: Whats up with the spacing on these lines? 30, 31, and 38 are indented more than 32-37

**Review by angel12 — CHANGES_REQUESTED** [review]
left some comments for you, please have a look at them

File: example.py (hunk 1 above)
Comment (craigjmidwinter) [reply example.py:6]: Good catch! I'll update

File: example.py (hunk 2 above)
Comment (craigjmidwinter) [reply example.py:31]: Hmm, not sure I see the difference here

Comment (craigjmidwinter) [reply example.py:31]: oh weird, I see it when I expand the diff, but it looks ok in my editor. I'll have a look

Comment (craigjmidwinter) [reply example.py:31]: hmm weird. Not sure what's up here-- everywhere else this looks ok, even in the committed version when I click on view file on this

**Pull request merged**